news/2018-08-18-aspire-2018-begins.md

```diff
@@ -0,0 +1,68 @@
+* **Any entries with additional resources (hitsounds, storyboards, etc) must be submitted in .osz format.** There are no size restrictions this time around, but if your .osz file does not get accepted by the contest system for being too large, PM [Ephemeral](http://osu.ppy.sh/u/102335) via the forums for assistance.
+* **The drain time of your entry must be at least 70% of the song's total length.** Basically, you have to map at LEAST 70% of the song. I recommend you map all of it.
+
+All entries will be judged by a panel of expert community members selected from across the beatmapping scene and reduced to roughly 12 in total before being put up for community voting after a stream feature. The winning sets will then be decided based on a combination of double-blind community voting and blind scoring from the panelists.
+
+Given the rising presence of "spectacle" entries into the Aspire contests, I will also be introducing a new scoring category revolving around visual impression, aka, spectacle. 
+
+Each set will be awarded a visual spectacle score by the panelist judges that is completely separate from all other scoring, and the set with the highest spectacle score at the end of judging will receive a special prize. Anything that influences how the map visually appears (aka, storyboarding, art, hitsounds) will count towards this score.
+
+Visuals will remain an important aspect of Aspire, but this new scoring should allow for more focus on the beatmapping aspect of creativity for those not so visually inclined. The super-high achievers among you may also note that the winning entry can also conceivably score 1st place in the normal criteria, and also win the spectacle voting for two prizes in one.
+
+The prizes are as follows:
+
+* **1st place:** 6 months of osu!supporter, unique Aspire 2018 profile badge, osu!news showcase featuring their winning entry and an *Elite Mapper: Aspirant* forum title (or special incrementation if they already have an Aspirant/Elite Mapper title), mapset enshrined forevermore as an "Aspire" class Ranked map.
+* **Most Spectacular:** the set assessed to have the highest visual spectacle score by the panellists will win 3 months of osu!supporter, a mention in the announcement post for the results, and a unique profile badge.
+* **Honorable Mentions:** any standout entries as judged by the panellists will receive 1 month of osu!supporter, and a mention in the announcement post for the results, plus a profile badge detailing their triumph.
+
+Entries will remain open for the next **32 days** from the date of this post. [Consult the contest listing for a countdown timer if you're unsure.](https://osu.ppy.sh/community/contests/68).
+
+Use everything you've got. Storyboarding, hitsounding, timing points, if the game can do it, it's yours to use.
```

Comment:
maybe add text editing since imo that is where the highest potential is in aspire rn
optional tho